Collection of grammar and syntax changes (#13)
aseprite/docs: sprite.md

In Aseprite a document/file/sprite has the following properties:

1. It has a [size](sprite-size.md) in pixels (width and height).
1. It has a [color mode](color-mode.md), which tells you how many
   colors the image can handle. All images in the sprite are in one
   specific mode, you cannot mix RGB images with Indexed images in the
   same sprite.

> 1. A [color profile](color-profile.md) which says what color space the sprite's RGB values will use.

Reviewer: We can say:
```suggestion
1. A [color profile](color-profile.md) which indicates what color space the RGB values are in.
```
